src/wazuh_db/wdb_parser.c

```diff
@@ -5235,22 +5226,6 @@ int wdb_parse_global_insert_agent_belong(wdb_t * wdb, char * input, char * outpu
     return OS_SUCCESS;
 }
 
-int wdb_parse_global_delete_group_belong(wdb_t * wdb, char * input, char * output) {
```

Comment:
If we are removing `wdb_parse_global_delete_group_belong()`, it would be cleaner to also remove all its UT.

Reply:
They are removed now.